content/docs/howto/token.md

```diff
@@ -24,10 +24,10 @@ Except any function of your token itself contains more than asset, we recommend
 
 ## How to mint
 
-### 1. Using deployed token (recommended)
+### 1. Using deployed token factory (recommended)
 
-The standard CW20 token is already stored in Terra network as code ID `3`.\
-The code is wrapped based on cw20 [v0.8.0](https://docs.rs/cw20/0.8.0/cw20/index.html) \
+The standard CW20 token is already stored in Terra 2.0 network as code ID `4`, and Terra classic as `3`.\
+The code is wrapped based on cw20 [v0.11.0](https://docs.rs/cw20/0.11.0/cw20/index.html) \
```

Comment:
We used 0.13.2

Reply:
https://github.com/DELIGHT-LABS/docs-terraswap-io/pull/35/commits/3af6325c4cbb4873cc7f33efa9a0b0a00d3efb9f